core/payment-driver/src/processor.rs

```diff
@@ -0,0 +1,76 @@
+use crate::PaymentDriver;
+use crate::PaymentDriverResult;
+use chrono::{DateTime, Utc};
+use std::sync::Arc;
+use ya_client_model::NodeId;
+use ya_core_model::driver::{
+    AccountBalance, AccountMode, Balance, PaymentAmount, PaymentConfirmation, PaymentDetails,
+    PaymentStatus,
+};
+use ya_persistence::executor::DbExecutor;
+
+#[derive(Clone)]
+pub struct PaymentDriverProcessor {
```

Comment:
This struct seems redundant. `db_executor` field is never used. And if we remove this field, we're left with just an `Arc` holding a `PaymentDriver`.

Reply:
:+1: